src/firefly/js/templates/fireflyviewer/TriViewPanel.jsx

```diff
@@ -25,7 +25,7 @@ import {
 } from '../../visualize/ui/TriViewImageSection.jsx';
 import {ResultsPanel} from './ResultsPanel.jsx';
 
-const stateKeys= ['title', 'mode', 'showTables', 'showImages', 'showXyPlots', 'images', 'coverageSide'];
+const stateKeys= ['title', 'mode', 'showTables', 'showImages', 'showXyPlots', 'images', 'coverageSide', 'initLoadCompleted'];
```

Comment:
What is 'initLoadCompleted'?  It is not the same as onAppReady?

Reply:
`initLoadCompleted` is a boolean that is set in LayoutInfo after the first table or image or plot has loaded. It been around for three or four years.

The problem was that TriViewPanel should have gotten it from `getLayoutInfo()` with the other stuff instead of relying on a stale property. I am not sure how it worked before. 

As I think about it, maybe it would be better for TriViewPanel just to maintained a static boolean. I will think about it.